packages/pds-ember/addon/components/pds/cta-link/docs.mdx

```diff
@@ -0,0 +1,54 @@
+import { Props } from '@storybook/addon-docs/blocks';
+export const TITLE = 'Components / CtaLink';
+
+# CTA Link
+The CTA Link component extends the Ember `<LinkTo>` component by applying
+additional CSS class names in order to give the resulting hyperlink a
+button-like appearance.
+
+
+- [Usage](#usage)
+- [Styling](#styling)
+- [Markup](#markup)
+- [Accessibility](#accessibility)
+- [See Also](#see-also)
+
+
+## Usage
+`<Pds::CtaLink />` has the same functionality as the
+[`<LinkTo />`](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/LinkTo?anchor=LinkTo)
+Ember component.
+
+**NOTE:** `<Pds::CtaLink>` renders a hyperlink (`<a href="...">`), which cannot
+be semantically `:disabled`.
+
+### @disabled
```

Comment:
Could you list all attributes in this doc? I see `route` in the usage below, not sure what others there might be. I'd find it helpful to have a list in one place

Reply:
I'd rather not, because those attributes are controlled by the `<LinkTo>` component in Ember.  I've included a link to the `<LinkTo>` documentation above, but Storybook styles don't make it obvious that it's a link.